src/rickbox/database/__init__.py

```diff
@@ -0,0 +1,21 @@
+from database.database_helper import DatabaseHelper
```

Comment:
Module that adds quality of life stuff. Should this be in DatabaseHelper?

Reply:
I don't like the idea of stuff in the `__init__.py` but  don't know Python best practices on this. I personally think the errors should be in `errors.py` or something.